pkg/udf/map_udf.go

```diff
@@ -130,9 +132,11 @@ func (u *MapUDFProcessor) Start(ctx context.Context) error {
 	}
 
 	maxMessageSize := sharedutil.LookupEnvIntOr(dfv1.EnvGRPCMaxMessageSize, sdkclient.DefaultGRPCMaxMessageSize)
-
 	enableMapUdfStream := sharedutil.LookupEnvBoolOr(dfv1.EnvMapStreaming, false)
+	enableBatchMapUdf := sharedutil.LookupEnvBoolOr(dfv1.EnvBatchMap, false)
```

Comment:
This options should be mutually exclusive, check it somewhere.

Reply:
Done.